vSync/vs_server.lua

```diff
@@ -81,10 +87,16 @@ RegisterCommand('freezetime', function(source, args)
         else
             print("Time is no longer frozen.")
         end
+		TriggerClientEvent('vSync:itemFreezeTimeSync', -1, freezeTime)
     end
 end)
 
-RegisterCommand('freezeweather', function(source, args)
+RegisterCommand('freezeweather', function(source)
+	TriggerClientEvent('vSync:freezeWeather', source)
```

Comment:
Same here, seems like an unnecessary client event to me, unless I'm missing something obvious here.